Doc/tools/extensions/pyspecific.py

```diff
@@ -58,11 +59,33 @@
 
 def issue_role(typ, rawtext, text, lineno, inliner, options={}, content=[]):
     issue = utils.unescape(text)
+    # sanity check: there are no bpo issues within these two values
+    if int(issue) > 47261 and int(issue) < 400000:
+        msg = inliner.reporter.error(f'The bpo id {text!r} seems too high -- '
+                                     f'use :gh:`...` for GitHub ids.', line=lineno)
+        prb = inliner.problematic(rawtext, rawtext, msg)
+        return [prb], [msg]
     text = 'bpo-' + issue
     refnode = nodes.reference(text, text, refuri=ISSUE_URI % issue)
     return [refnode], []
 
 
+# Support for marking up and linking to GitHub issues
+
+def gh_issue_role(typ, rawtext, text, lineno, inliner, options={}, content=[]):
+    issue = utils.unescape(text)
+    # sanity check: all GitHub issues have highest ID
+    # note that there is some overlapping with bpo ids
+    if int(issue) < 32427:
```

Comment:
This is actually 32426, not 32427.
```suggestion
    if int(issue) < 32426:
```